ansible_collections/arista/avd/plugins/plugin_utils/cv_client/client/change_control.py

```diff
@@ -169,3 +181,43 @@ async def start_change_control(
 
         except Exception as e:
             raise get_cv_client_exception(e, f"Change Control ID '{change_control_id}'") or e
+
+    async def wait_for_change_control_complete(
+        self: CVClient,
+        cc_id: str,
+        state="COMPLETED",
```

Comment:
I don't think we want a default here.
```suggestion
        state: Literal["completed", "unspecified", "running", "scheduled"]
```